vue.config.js

```diff
@@ -1,21 +1,12 @@
 module.exports = {
-  crossorigin: 'anonymous',
+  crossorigin: "anonymous",
```

Comment:
Why all single quotes are replaced with double?

Reply:
I'm so sorry. Prettier has double quotes by default. Changed it in the settings. I'll try to check the code better.